aten/src/ATen/native/quantized/cpu/fake_quantize_per_channel_affine.cpp

```diff
@@ -0,0 +1,148 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+#include "fake_quantize_core.h"
+
+/* FakeQuantize Op for PerChannelAffine quantization scheme */
+namespace at {
+namespace native {
+
+/* Per channel fake-quantizes the 'inputs' tensor.
+Args:
+  X: Forward input tensor.
+  dY: Backward input tensor (_backward op only).
+  scale: scale of per channel affine quantization
+  zero_point: zero_point of per channel affine quantization
+  axis: int specifying the axis to be quantized
+  quant_min: minimum quantized value
+  quant_max: maximum quantized value
+Returns:
+  Fake quantized tensor (double dtype).
+
+*/
+
+Tensor fake_quantize_per_channel_affine_cpu(
+    const Tensor& self,
+    const Tensor& scale,
+    const Tensor& zero_point,
+    int64_t axis,
+    int64_t quant_min,
+    int64_t quant_max) {
+
+  TORCH_CHECK(self.scalar_type() == ScalarType::Float);
+
+  TORCH_CHECK(scale.size(0) == zero_point.size(0),
```

Comment:
nit: after the `dim() == 1` check, this can be `scale.numel()` == `zero_point.numel()`